### PR TITLE
typescript-conditionals

### PR DESCRIPTION
**Pull request by TristanChao**
Screenshot typescript-conditionals

### CODE REVIEW

File: typescript-conditionals/main.ts [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+    return true;
+  } else {
+    return false;
+  }
+}
+
+function isOldEnoughToDrinkAndDrive(person: Person): boolean {
+  if (person.age > 0) {
+    return false;
+  } else {
```
Comment (brandon-moy) [inline typescript-conditionals/main.ts:59]: Can a person drink or drive if they are just older than 0?

File: typescript-conditionals/main.ts [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+
+function categorizeAcidity(pH: number): string {
+  if (pH >= 0 && pH < 7) {
+    return 'acid';
+  } else if (pH >= 7 && pH <= 14) {
+    return 'base';
+  } else {
+    return 'invalid pH level';
+  }
+}
```
Comment (brandon-moy) [inline typescript-conditionals/main.ts:72]: What about a neutral level of acidity?

**Review by brandon-moy — CHANGES_REQUESTED** [review]
Couple of functions with changes requested, rest of it is looking good though 👍

File: typescript-conditionals/main.ts (hunk 1 above)
Comment (TristanChao) [reply typescript-conditionals/main.ts:59]: I originally made the function return false right away but it gave me an error for not using the `person` parameter. I wrote the `if` statement to use the parameter and still return false no matter what.

Comment (brandon-moy) [reply typescript-conditionals/main.ts:59]: While yes, it is true that the function should return `false` all of the time, the function should still do what it is intended to do: check if the person is old enough to drink and if the person is old enough to drive.

**Review by brandon-moy — APPROVED** [review]
Approved! 👍

**Pull request merged**